examples/rust/tutorial/alloc_u8.rs

```diff
@@ -0,0 +1,15 @@
+use std::alloc::{alloc, dealloc, handle_alloc_error, Layout};
+
+unsafe fn main()
+//@ req true;
+//@ ens true;
+{
+    let l = Layout::new::<u8>();
+    let p = alloc(l);
+    if p.is_null() {
+        handle_alloc_error(l);
+    }
+    *p = 42;
+    assert!(*p == 42);
```

Comment:
Does VeriFast support the code that this macro expands to? See how I define an assert function in the examples. Or maybe we should add support for this macro.

Reply:
Yes! Fortunately, it does.